accelerometer/summariseEpoch.py

```diff
@@ -13,13 +13,13 @@
 from datetime import timedelta
 
 def getActivitySummary(epochFile, nonWearFile, summary,
```

Comment:
Nothing to see here. This is just a quirk of my editor (VSCode) that automatically removes trailing spaces (a best practice) in the file.